AngularJS/app.js

```diff
@@ -0,0 +1,214 @@
+angular.module("reportApp", ['main']);
+var app = angular.module("main", []);
+
+angular.module('reportApp')
+    .service('reportService', ReportService)
+    .component('reportList', {
+        templateUrl: 'reportList.html',
+        controller: ReportListController
+    })
+
+function ReportListController(reportService) {
+    //Lists all reports in tabular form, with edit and remove options against each report item.
```

Comment:
Having no comment is better than a misleading one.
Update the comment to include that this also adds new reports.